massive-data-import/README.md

```diff
@@ -0,0 +1,152 @@
+# Massive Data Import to Kuzzle with Geofencing subscription
+
+## Introduction
+
+When processing large volumes of data with Kuzzle, you may need to import large datasets quickly.  
+
+Kuzzle offers two massive data import systems according to your needs.  
+In some cases, you only want to quickly import data into the database to make subsequent queries on and in others you need all the features (Real-Time notifications, ...) of Kuzzle also during import.
+
+In this How-To, we will explore the two massive import techniques of Kuzzle.
+For this example we will use data from the NYC Yellow Taxi dataset.  
+
+## Architecture
+
+We will be using the standard Kuzzle stack (Kuzzle, Elasticsearch and Redis). (Check [docker-compose.yml](docker-compose.yml) for more details)
+
+On Kuzzle, the data will be stored in the `yellow-taxi` collection of the `nyc-open-data` index according to the following mapping:
+
+```js
+{
+  "pickup_datetime":  { "type": "date", "format": "MM/dd/yyyy hh:mm:ss a" },
+  "dropoff_datetime": { "type": "date", "format": "MM/dd/yyyy hh:mm:ss a" },
+  "passenger_count":  { "type": "long" },
+  "trip_distance":    { "type": "double" },
+  "pickup_position":  { "type": "geo_point" },
+  "dropoff_position": { "type": "geo_point" },
+  "fare_amount":      { "type": "double" }
+}
+```
+
+## Importing data to Kuzzle
+
+The first way to import important data sets into Kuzzle is to use the [Bulk Controller](https://docs.kuzzle.io/api-documentation/controller-bulk/) [import action](https://docs.kuzzle.io/api-documentation/controller-bulk/import/).
+Its operation and syntax is similar to that of the [Elasticsearch Bulk API](https://www.elastic.co/guide/en/elasticsearch/reference/5.5/docs-bulk.html).
+
+This method is very fast but it writes almost directly in Elasticsearch. Other Kuzzle features such as [Real-Time Notifications](https://docs.kuzzle.io/guide/essentials/real-time/) will not be available.  
```

Comment:
edit: ok I just saw that you put a few numbers at the end of this document. I think that a real production environment should be used instead of a laptop to get these numbers (because a back-end will almost never run on a laptop in real-life situations), and the result should be shown around here instead of at the end of this how-to